plugins/auth-backend/api-report.md

```diff
@@ -107,6 +108,15 @@ export interface AuthProviderRouteHandlers {
   start(req: express.Request, res: express.Response): Promise<void>;
 }
 
+// Warning: (ae-missing-release-tag) "AuthResolverContext" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
```

Comment:
i'd appreciate if you could put a doc comment on these types that go to the actual public interface, and have a `@public` in them - so that the API reports become nice and clean

(same below too)

Reply:
Done 👍

Does it look good to you?